traffic_ops/v5-client/serviceCategory.go

```diff
@@ -28,24 +28,26 @@ import (
 const apiServiceCategories = "/service_categories"
 
 // CreateServiceCategory creates the given Service Category.
-func (to *Session) CreateServiceCategory(serviceCategory tc.ServiceCategory, opts RequestOptions) (tc.Alerts, toclientlib.ReqInf, error) {
+// tc.ServiceCategoryV5 used RFC3339 Time Format
```

Comment:
I don't think we need to call that out here. It's a different struct, and it can be different in any way in the future, not just in timestamp JSON encoding format.

Reply:
Removed the comments